packages/im-base/src/constants/mockDBResponse.json

```diff
@@ -0,0 +1,128 @@
+      "url": "http://www.youtube.com/watch?v=DLX62G4lc44",
+      "thumbnails": {
+        "default": {
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/default.jpg",
+          "width": 120,
+          "height": 90
+        },
+        "medium": {
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/mqdefault.jpg",
+          "width": 320,
+          "height": 180
+        },
+        "high": {
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/hqdefault.jpg",
+          "width": 480,
+          "height": 360
+        },
+        "standard": {
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/sddefault.jpg",
+          "width": 640,
+          "height": 480
+        },
+        "maxres": {
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/maxresdefault.jpg",
+          "width": 1280,
+          "height": 720
+        }
+      },
+      "genre": "Thriller",
+      "hotspots": [
+        {
+          "id": 1,
+          "name": "Test",
```

Comment:
update mockData to contain hotspots. overlays and triggers as objects and not lists

Reply:
we won't need mock data any more for this one